src/Mado/QueryBundle/Queries/OrFilterFactory.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace Mado\QueryBundle\Queries;
+
+use Mado\QueryBundle\Services\StringParser;
+
+class OrFilterFactory
+{
+    private const OR_OPERATOR_LOGIC = 'OR';
+
+    private $entityAlias;
+
+    private $fields;
+
+    private $joinFactory;
+
+    private $parser;
+
+    private $conditions;
+
+    private $parameters;
+
+    private $relationEntityAlias;
+
+    public function __construct(string $entityAlias, array $fields, JoinFactory $joinFactory)
+    {
+        $this->entityAlias = $entityAlias;
+        $this->joinFactory = $joinFactory;
+        $this->fields = $fields;
+        $this->conditions = '';
+        $this->parameters = [];
+        $this->parser  = new StringParser();
+    }
+
+    public function createFilter(array $orFilters)
```

Comment:
Could I suggest `OrFilter::create(arra $filters)`?

Reply:
I don't know if now is time to rendere static the function create I think